### PR TITLE
added comments to file for understanding

### PR DESCRIPTION
**Pull request by TantorskiJake**

### CODE REVIEW

File: src/services/database/models/user.js [hunk 1]
```diff
@@ -30,6 +30,13 @@ const UserSchema = new Schema(
       required: true,
       enum: ['male', 'female'],
     },
+    // Including gender as a field in the user model might be unnecessary:
```
Comment (JWilson45) [inline src/services/database/models/user.js:33]: If this is your argument, would you consider removing phone number like we removed address? Phone number could be used as a login field but it's not necessary

**Review by JWilson45 — COMMENTED** [review]
Should we remove this and more so we don't deal with sensitive data?

File: src/services/database/models/user.js (hunk 1 above)
Comment (TantorskiJake) [reply src/services/database/models/user.js:33]: I would say only keep it if we are gonna use it for login or 2FA. I think we remove address and just keep zip code. Only need fields that would be relevant metadata

Comment (TantorskiJake) [reply src/services/database/models/user.js:33]: I think the less fields the better on the models to start then we can add as needed

**Review by JWilson45 — APPROVED** [review]

**Pull request merged**